src/local/room_name.rs

```diff
@@ -23,6 +40,8 @@ pub struct RoomName {
     ///
     /// This is the same representation of the upper 16 bits of [`Position`]'s
     /// packed representation.
+    ///
+    /// [`Position`]: crate::local::Position
```

Comment:
Oh, great thanks, I saw the doc compiler complaining about that earlier!

Reply:
:+1: I almost made a separate PR but decided just to include it in this one